ISICbalance/ViewControllers/AccountViewController.swift

```diff
@@ -66,7 +97,15 @@ class AccountView: UIView {
         }
     }
     
-    required init?(coder aDecoder: NSCoder) {
-        fatalError("init(coder:) has not been implemented")
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        navigationItem.title = L10n.Login.title
+        loginButton.addTarget(self, action: #selector(saveCredentials), for: .touchDown)
+    }
+    
+    @objc func saveCredentials() {
+        viewModel.saveCredentials(username: usernameTextField.text ?? "", password: passwordTextField.text ?? "")
```

Comment:
❔ zvládnul bys to napsat reaktivně? Ve středu na cviku budu něco podobného dělat, kdyžtak vydrž tam a uvidíš to na vlastní 👀

Reply:
tedy aby se hodnoty z textFieldu automaticky během psaní propisovaly do viewModelu a při kliku na tlačítko už by byly ve viewModelu? Současně s tím můžeš mít pak nastavený `loginButton.isEnabled` podle toho, jestli jsou obě hodnoty vyplněné nebo ne :)

Reply:
Myslím si, že zvládl 😁 Na reaktivním swiftu pracuji v další branchi. Zatím jsem tam udělal requesty podle cvik. Tohle tam tedy doimplementuji také.

Reply:
🆒